coralproject/talk
# [CORL-1747] Live Chat: Merge the root and conversation "comment entered" subscriptions

## What does this PR do?

Merges the live chat root and conversation "comment entered" subscriptions together to prevent race conditions

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Use live chat
- Create comments
- See that it updates the "jump to new", live insertion, etc on both the main chat and also the conversation views
